src/Alert/Alert.scss

```
@@ -25,7 +25,7 @@
   .alert-icon {
     float: left;
     margin-right: $alert-icon-space;
-    width: 1.25rem;
+    width: $alert-icon-width;
     height: 1.25rem;
```

```suggestion
    height: $alert-icon-width;
```

Icons are square so the width and height should match in these styles.